core/src/main/java/org/bitcoin/crawler/PeerSeedProtos.java

```diff
@@ -3,6 +3,7 @@
 
 package org.bitcoin.crawler;
 
+@SuppressWarnings("ALL")
```

Comment:
Ah ok I see. But if they are generated, your annotations will be gone soon? I think we'd need to find a way so that the source file contains the annotation, or an instruction to generate the annotation.